### ISSUE TITLE
kanbn task -i ends with "instance.due is not of a type(s) date"

### ISSUE THREAD

**Issue by Jc-L** [basementuniverse/kanbn#3]
Adding a task interactively with a `Due date: 2022/11/26` ended up with error

```
Unable to build task:
instance.due is not of a type(s) date
```

Error is thrown from a call to `validateMetadataFromJSON()` due to `due` being a string there:

```
{ due: '2022-11-03T13:48:48.155Z', created: 2022-11-03T13:48:53.003Z }
```

Not sure if this validation should not be a `validateMetadataFromMarkdown()`, I tested and it seems to work.

kanbn version: v0.10.0
Location is non-US

**Comment by basementuniverse** [comment]
Good catch, this is now fixed in https://github.com/basementuniverse/kanbn/releases/tag/1.0.0
